views/js/pciCreator/ims/audioRecordingInteraction/runtime/audioRecordingInteraction.js

```diff
@@ -319,6 +319,34 @@ define([
                 this.initBeepPlayer();
                 this.updateResetCount();
                 this.initRecording();
+
+                if (this.config.enableDomEvents) {
```

Comment:
Why not always enable them, without a flag?
It's fine to leave this, just wondering.